### ISSUE TITLE
Add keyword search function to admin search pages

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#3247]
Mentors indicate their school or company name using an open text field. This makes searching for all the mentors or judges from a certain organization challenging as they may have entered slightly different names (i.e. Google, Google Brazil, Google (retired), etc). 

It would be useful if admins could keyword search for mentors or judges and have all school or company names that include that keyword return in results. Currently, admins must select a company from a drop-down list when searching for mentors or judges.

**Comment by viviancan** [comment]
@dboyer Verifying that this search should only return results for mentor and judge accounts?

**Comment by dboyer** [comment]
Correct. Program team confirmed that this search should only apply to mentors and judges, not students.
